pkg/project/project.go

```diff
@@ -1,7 +1,7 @@
 package project
 
 var (
-	bundleVersion        = "6.4.0"
+	bundleVersion        = "6.6.0"
```

Comment:
Do we have versions `6.1-5.x` or we start v6 with `6.6.0`?

Reply:
This is all only for testing. We have to figure out the Node Pool versions this week. This PR will be closed soon without merging.

Reply:
Ack.